rest/taskana-rest-spring/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java

```diff
@@ -235,13 +234,13 @@ void testReturn400IfCreateClassificationWithIncompatibleParentIdAndKey() throws
                     new HttpEntity<>(newClassification, restHelper.getHeaders()),
                     ParameterizedTypeReference.forType(ClassificationResource.class)));
 
-    assertNotNull(e);
-    assertEquals(HttpStatus.BAD_REQUEST, e.getStatusCode());
+    assertThat(e).isNotNull();
```

Comment:
We can substitute these Assertions.assertThrows cases with assertJ as follows: 

    assertThatThrownBy(

        () -> template.exchange(
            restHelper.toUrl(Mapping.URL_CLASSIFICATIONS),
            HttpMethod.POST,
            new HttpEntity<>(newClassification, restHelper.getHeaders()),
            ParameterizedTypeReference.forType(ClassificationResource.class)))
        .isInstanceOf(HttpClientErrorException.class)
        .extracting(ex ->((HttpClientErrorException) ex ).getStatusCode())
          .isEqualTo(HttpStatus.BAD_REQUEST);